ecoscope_workflows/tasks/results/_map_config.py

```diff
@@ -0,0 +1,32 @@
+from pydantic import BaseModel
+
+
+class LayerStyleProperty(BaseModel):
+    auto_highlight: bool = False
+    opacity: float = 1
+    pickable: bool = True
+
+
+class PathLayerProperty(LayerStyleProperty):
```

Comment:
```suggestion
class PolylineLayerProperty(LayerStyleProperty):
```
this layer has been renamed in the latest `ecoscope`